Assets/MRTK.ThirdParty/MRTK-Quest/Editor/MRTKQuestConfigurationChecker.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace prvncher.MixedReality.Toolkit.OculusQuestInput
+{
+    /// <summary>
+    /// Class that checks if the Oculus Integration Assets are present and configures the project if they are.
+    /// </summary>
+    [InitializeOnLoad]
+    static class OculusConfigurationChecker
+    {
+        // The presence of the OculusProjectConfig.asset is used to determine if the Oculus Integration Assets are in the project.
+        private const string OculusIntegrationProjectConfig = "OculusProjectConfig.asset";
+        private static readonly string[] Definitions = { "OCULUSINTEGRATION_PRESENT" };
+
+        static OculusConfigurationChecker()
+        {
+            // Check if Oculus Integration Package is in the project
+            ReconcileOculusIntegrationDefine();
+        }
+
+        //seems to work fine as is
+        private static bool ReconcileOculusIntegrationDefine()
+        {
+            FileInfo[] files = FindFilesInAssets(OculusIntegrationProjectConfig);
+
+            if (files.Length > 0)
+            {
+                AppendScriptingDefinitions(BuildTargetGroup.Android, Definitions);
```

Comment:
Note that you should add this to the windows standalone build target as well, for Rift and Rift S users.